test/integration/crud/explain.test.ts

```diff
@@ -0,0 +1,282 @@
+import { expect } from 'chai';
+
+import {
+  type Collection,
+  type CommandStartedEvent,
+  type Db,
+  type MongoClient,
+  MongoServerError
+} from '../../mongodb';
+
+describe('Explain', function () {
+  let client: MongoClient;
+  let db: Db;
+  let collection: Collection;
+  let commandsStarted: CommandStartedEvent[];
+
+  beforeEach(async function () {
+    client = this.configuration.newClient({ monitorCommands: true });
+    db = client.db('queryPlannerExplainResult');
+    collection = db.collection('test');
+    commandsStarted = [];
```

Comment:
(nitpick) since this is only used in one test, you can move the declaration of `commandsStarted` and the setup of the `'commandStarted'` listener into that test.